databricks/koalas/series.py

```diff
@@ -1913,6 +1914,133 @@ def _fillna(self, value=None, method=None, axis=None, limit=None, part_cols=()):
             )
         )._kser_for(self._column_label)
 
+    def factorize(self, sort: bool = True, na_sentinel: Optional[int] = -1):
```

Comment:
Shall we add the return type annotation?

Reply:
Sounds good!